bin/loopDHS

```diff
@@ -25,9 +27,16 @@ from pydhsfw.automl import AutoMLPredictRequest, AutoMLPredictResponse
 from pydhsfw.jpeg_receiver import JpegReceiverImagePostRequestMessage
 from pydhsfw.axis import AxisImageRequestMessage, AxisImageResponseMessage
 
-#_logger = logging.getLogger(__name__)
+# logging setup
```

Comment:
Better to remove commented out lines of code when checking in. You can always easily get them back using history if you need them.

Reply:
I added a small code comment just for the purposes of delineating the lines of code where logging is being configured.

Reply:
Riiiiight.... I missed that. Oops. Thumbs up!